include/cose/cose_configure.h.in

```diff
@@ -1,20 +1,39 @@
 //
-//  Determine which cryptographic library we are going to be using
+// cose_configure.h. Generated from cose_configure.h.in by CMake
+// Determine which cryptographic library we are going to be using
 //
 
 #pragma once
 
-#if defined(USE_MBED_TLS)
-#if defined(USE_OPEN_SSL) || defined(USE_BCRYPT)
+#cmakedefine COSE_C_USE_MBEDTLS
+#cmakedefine COSE_C_USE_OPENSSL
+#cmakedefine COSE_C_USE_BCRYPT
+
+// make sure we only use one crypto lib
+#if defined(COSE_C_USE_MBEDTLS)
+#if defined(COSE_C_USE_OPENSSL) || defined(COSE_C_USE_BCRYPT)
 #error Only Define One Crypto Package
 #endif
-#elif defined(USE_BCRYPT)
-#if defined(USE_OPENSSL)
```

Comment:
@jimsch there was `USE_OPENSSL` and `USE_OPEN_SSL`

Reply:
Either is fine